### PR TITLE
[MIG] website_crm_privacy_policy: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
- Update manifest file
- Update README file

cc @Tecnativa

### CODE REVIEW

File: website_crm_privacy_policy/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014 Jorge Camacho <jcamacho@trey.es>
+# Copyright 2015 Antonio Espinosa <antonioea@antiun.com>
+# Copyright 2016-2018 Vicent Cubells <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Website CRM privacy policy",
+    "category": "Website",
+    "version": "11.0.1.0.0",
+    "website": "http://www.trey.es",
```
Comment (rafaelbn) [inline website_crm_privacy_policy/__manifest__.py]: This should be https://odoo-community.org/

File: website_crm_privacy_policy/README.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Thanks to:
+* [johnny_automatic](https://openclipart.org/user-detail/johnny_automatic)
+* [jetxee](https://openclipart.org/user-detail/jetxee)
+
+
+Contributors
+------------
+
+* Jorge Camacho <jcamacho@trey.es>
+* Antonio Espinosa <antonioea@antiun.com>
```
Comment (rafaelbn) [inline website_crm_privacy_policy/README.rst]: antonio.espinosa@tecnativa.com

File: website_crm_privacy_policy/README.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+* Rafael Blasco <rafabn@antiun.com>
```
Comment (rafaelbn) [inline website_crm_privacy_policy/README.rst]: rafael.blasco@tecnativa.com

File: website_crm_privacy_policy/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply website_crm_privacy_policy/__manifest__.py]: It's better to put the repository, but you can leave it this time: https://github.com/OCA/website.

**Review by pedrobaeza — APPROVED** [review]
Tried on runbot and working without problems.

File: website_crm_privacy_policy/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014 Jorge Camacho <jcamacho@trey.es>
+# Copyright 2015 Antonio Espinosa <antonioea@antiun.com>
+# Copyright 2016-2018 Vicent Cubells <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Website CRM privacy policy",
+    "category": "Website",
+    "version": "11.0.1.0.0",
+    "website": "https://odoo-community.org/",
```
Comment (chienandalu) [inline website_crm_privacy_policy/__manifest__.py]: https://github.com/OCA/website

**Review by chienandalu — APPROVED** [review]
Non blocking comment.

**Pull request merged**